packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -37,15 +18,23 @@ const LinkControlSearchInput = ( {
 	fetchSuggestions,
 	onReset,
 	showInitialSuggestions,
+	errorMsg,
 } ) => {
 	const [ selectedSuggestion, setSelectedSuggestion ] = useState();
 
+	/**
+	 * Handles the user moving between different suggestions. Does not handle
+	 * choosing an individual item.
+	 *
+	 * @param {string} selection the url of the selected suggestion
+	 * @param {Object} suggestion the suggestion object
+	 */
 	const selectItemHandler = ( selection, suggestion ) => {
 		onChange( selection );
 		setSelectedSuggestion( suggestion );
 	};
 
-	function selectSuggestionOrCurrentInputValue( event ) {
+	function handleSubmit( event ) {
```

Comment:
Not sure if this was a casualty of rebase, but the previous name did better to explain what exactly we expect to be handled, in absence of some more thorough JSDoc explanation.

Reply:
Agreed. I think this was a rebase issue (again).